SDBlockDevice.cpp

```diff
@@ -344,6 +344,11 @@ int SDBlockDevice::init()
     }
     debug_if(SD_DBG, "init card = %d\n", _is_initialized);
     _sectors = _sd_sectors();
+    // CMD9 failed
+    if (0 == _sectors) {
```

Comment:
Not really, if initialization of SD card fails cannot be recovered without reset / replug. 

However, in new driver other recovery methods are implemented like retry command (If no response from card / card busy).  _sd_sectors() -> _cmd () [This function does retry 3 times for every command, till we get positive response from card).

Error here was actually from the code, device response was fast and we missed start token while freeing SPI bus. Even in case of another trial we will miss the start token.